include/libtorrent/piece_picker.hpp

```diff
@@ -97,9 +97,9 @@ namespace libtorrent {
 			// priority factor
 			prio_factor = 3,
 			// max blocks per piece
-			// there are counters in downloading_piece that only have 15 bits to
+			// there are counters in downloading_piece that only have 16 bits to
```

Comment:
I didn't see that the counter itself was changed. Just this comment... Or is it a 16-bit size already?

Reply:
`num_blocks` went from 14 bits to a `uint16_t` (i.e. 16 bits)

Reply:
> `num_blocks` went from 14 bits to a `uint16_t` (i.e. 16 bits)

Ah, I thought it was about something else.